.azure-pipelines/templates/jobs/packaging-jobs.yml

```diff
@@ -50,6 +51,7 @@ jobs:
           py -3 -m venv venv
           venv\Scripts\python tools\pip_install.py -e certbot-ci
         displayName: Prepare Certbot-CI
+      # If we change the output filename from `certbot-beta-installer-win32.exe`, it should also be changed in tools/create_github_release.py
```

Comment:
Sorry I gave you a bad pointer here. The filename was set earlier in`windows-installer/construct.py`. I think it'd be nice to add a comment about this there.